src/utilities.js

```diff
@@ -38,7 +38,7 @@ export function isBlock (node) {
 
 export var voidElements = [
   'AREA', 'BASE', 'BR', 'COL', 'COMMAND', 'EMBED', 'HR', 'IMG', 'INPUT',
-  'KEYGEN', 'LINK', 'META', 'PARAM', 'SOURCE', 'TRACK', 'WBR'
+  'KEYGEN', 'LINK', 'META', 'PARAM', 'SOURCE', 'TRACK', 'WBR', 'TEXTAREA'
```

Comment:
The list is apparently sorted alphabetically. TEXTAREA should be between SOURCA and TRACK.